### DISCUSSION TITLE
Converting .ome.tiff to single channel tiffs

### DISCUSSION THREAD

**Discussion by jonhsussman** [angelolab/ark-analysis#717] [Q&A]
Hello! I am trying to implement this package with large multi-channel .ome.tiff. What is your suggestion for the best way to convert a single .ome.tiff file into the multiple single-channel tiff files that are required as input for the pixel clustering package. 

Thanks!

**Comment by ngreenwald** [comment]
We don't have an internal one we prefer, since the MIBI images come out as single-channel by default. I think @ackagel may have written something to go from single-channel to OME, not sure about the other way around?
